bin/run_xgettext.sh

```diff
@@ -1,9 +1,20 @@
 #!/bin/bash
-FULLPATH=$(dirname $(readlink -f "$0"))
+set -eo pipefail
 
-if [ "$1" == "--help" -o "$1" == "-h" ]
+function resolve {
+	if [ "$(uname)" == "Darwin" ]
```

Comment:
Uuuuuuugh.

Reply:
unfortunately, on linux `readlink -f` is the way to go, while `realpath` is deprecated (and looks like also in most of BSDs). But on osx `readlink` doesn't have `-f` flag, so the use of `realpath` is required. At least that's what I got looking the issue on "the internets"